File: PrepedidoBusiness/Dto/Prepedido/DetalhesPrepedido/PrepedidoProdutoDtoPrepedido.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PrepedidoBusiness.Dto.Prepedido.DetalhesPrepedido
{
    public class PrepedidoProdutoDtoPrepedido
    {
        public string Fabricante { get; set; }
        public string NumProduto { get; set; }
        public string Descricao { get; set; }
        public string Obs { get; set; }
        public short? Qtde { get; set; }
        public short Permite_Ra_Status { get; set; }
        public bool BlnTemRa { get; set; }
        public decimal? Preco { get; set; }
        public decimal VlLista { get; set; }
        public float? Desconto { get; set; }
        public decimal VlUnitario { get; set; }
        public decimal? VlTotalItem { get; set; }
        public decimal VlTotalRA { get; set; }
        public float? Comissao { get; set; }
        public decimal? TotalItemRA { get; set; }
        public decimal? TotalItem { get; set; }

    }
}

File: PrepedidoSpa/src/app/dto/prepedido/DetalhesPrepedido/PrepedidoProdutoDtoPrepedido.ts
export class PrepedidoProdutoDtoPrepedido {
    Fabricante: string;
    Fabricante_Nome: string;
    NormalizacaoCampos_Produto: string;
    Descricao: string;
    Obs: string;
    Qtde: number | null;
    Permite_Ra_Status: number;
    BlnTemRa: boolean;
    NormalizacaoCampos_CustoFinancFornecPrecoListaBase: number | null;
    NormalizacaoCampos_Preco_NF: number | null;
    NormalizacaoCampos_Preco_Lista: number;
    NormalizacaoCampos_Desc_Dado: number | null;
    NormalizacaoCampos_Preco_Venda: number;
    VlTotalItem: number | null;
    VlTotalRA: number;
    Comissao: number | null;
    TotalItemRA: number | null;
    TotalItem: number | null;
    AlterouValorRa: boolean | null;
    AlterouVlVenda: boolean | null;
    //verificar a necessidade dessa variavel
    Qtde_estoque_total_disponivel: number | null;
    ProdutoPai: string
    CustoFinancFornecCoeficiente : number |null;
}
